empathylink.lic

```diff
@@ -0,0 +1,118 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#empathylink
+=end
+
+custom_require.call(%w[common drinfomon equipmanager])
+
+class Empathylink
+  def initialize
+    if UserVars.empathylink.nil?
+      UserVars.empathylink=Hash.new
+    end
+    populate_noheal_list
+    settings = get_settings
+    @buddies = settings.hunting_buddies
+    @use_hodierna = DRC.get_settings.hashodierna
```

Comment:
If this is a new setting, it needs to be in `base.yaml`. I would also suggest renaming it to `has_hodierna`.

Would it be possible to do a room check or something instead?

Reply:
Right for adding to base.yaml. The @buddies pulls all your buddies in and then it only performs the link on those in the room.

Reply:
Added to base.

:question: Would it be possible to do a room check or something instead? It does a room check and compares room PCs to buddies list to touch.